rclrs/src/parameter/service.rs

```diff
@@ -247,46 +247,47 @@ impl ParameterService {
         // destruction is made for the parameter map.
         let map = parameter_map.clone();
         let describe_parameters_service = node.create_service(
-            &(fqn.clone() + "/describe_parameters"),
+            (fqn.clone() + "/describe_parameters").qos(QoSProfile::parameter_services_default()),
```

Comment:
rcl defines "default services profile" and "parameter services profile" as different pre-made QOS profiles. Tbh I haven't looked closely at whether they're different at all, but I would assume they're defined as different pre-made profiles just in case they need to be tuned differently in the future.

So `ServiceOptions` uses the "default service profile" as its default, and we override that with the "parameter services profile" for all the builtin parameter services.

Reply:
Ah missed that, yap that sounds more correct, thanks!